### ISSUE TITLE
Support for Opening IMAS H5s with OMAS?

### ISSUE THREAD

**Issue by allen-adastra** [gafusion/omas#319]
Hello!

We have tried opening `.h5` files that were generated for IMAS, but we have had trouble opening them with `omas.load_omas_h5`. Are we doing something wrong?

Thanks!

**Comment by orso82** [comment]
That's correct... actually, to be nit picky both the OMAS .h5 and .nc formats long predate the IMAS one 😉 

One way to support the native (tensorized) IMAS h5 file would be via `omasjl` (see https://github.com/gafusion/omas/issues/328#issuecomment-2613124814)
